site/content/how-to/traffic-management/client-settings.md

```diff
@@ -0,0 +1,283 @@
+---
+title: "Client Settings Policy"
+weight: 800
+toc: true
+docs: "DOCS-000"
+---
+
+Learn how to use the `ClientSettingsPolicy` API.
+
+## Overview
+
+The `ClientSettingsPolicy` API allows Cluster Operators and Application Developers to configure the connection behavior between the client and NGINX.
+
+The settings in `ClientSettingsPolicy` correspond to the following NGINX directives:
+
+- [`client_max_body_size`](<https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size>)
+- [`client_body_timeout`](<https://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_timeout>)
+- [`keepalive_requests`](<https://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_requests>)
+- [`keepalive_time`](<https://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_time>)
+- [`keepalive_timeout`](<https://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_timeout>)
+
+`ClientSettingsPolicy` is an [Inherited PolicyAttachment](https://gateway-api.sigs.k8s.io/reference/policy-attachment/) that can be applied to a Gateway, HTTPRoute, or GRPCRoute in the same namespace as the `ClientSettingsPolicy`.
+
+When applied to a Gateway, the settings specified in the `ClientSettingsPolicy` affect all HTTPRoutes and GRPCRoutes attached to the Gateway. This allows Cluster Operators to set defaults for all applications using the Gateway.
+
+When applied to an HTTPRoute or GRPCRoute, the settings in the `ClientSettingsPolicy` affect only the route they are applied to. This allows Application Developers to set values for their applications based on their application's behavior or requirements.
+Settings applied to an HTTPRoute or GRPCRoute take precedence over settings applied to a Gateway.
+
+TODO: link to general doc on policy inheritance
```

Comment:
Currently working on this doc as part of #1995. I'll remove this TODO if this is approved before I finish #1995.